ansible/examples/master.yml

```diff
@@ -36,11 +36,15 @@
       - 'postbuildscript'
       - 'jobConfigHistory'
 
-    - port: 8080
+    - jenkins_port: 8080
     - prefix: '/build'
     - xmx: 8192
```

Comment:
I know this was not set by you, but if this is doing what I *think* will do, then it will set the JAVA JVM max memory usage to 8GB of ram. At some point this worked, but I don't think you want that